### DISCUSSION TITLE
Cypress test works locally, fails in github-action with "Unexpected token"

### DISCUSSION THREAD

**Discussion by briansmith-fr** [cypress-io/cypress#14187] [CI setup]
I have a test that works locally, but fails in github-action: 
```
|   1) Create new record
|        Create the record is anybody home:
|      SyntaxError: The following error originated from your application code, not from Cypress.
| 
|   > Unexpected token '<'
| 
| When Cypress detects uncaught errors originating from your application it will automatically fail the current test.
| 
| This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.
| 
| https://on.cypress.io/uncaught-exception-from-application

```
I turned off listening to the exception as described, now I get a timeout waiting for a request that never occurred. 

But the request does occur.

What gotchas do I need to be on the lookout for when running locally vs running in github-actions?

**Comment by briansmith-fr** [comment]
I _think_ I'm starting to make progress:
1. The page is loading a `<script>` and was looking for its URL from the environment, which needed to be added to the YML in the workflow.
2. The page is loading google analytics; after adding the intercept as described, and adding `Cypress.on("window:before:load"` I'm getting a lot farther than before.
I think I'm getting close :-)

**Comment by bahmutov** [comment]
I suspect that one of the static resources returns HTML for some reason, which has `<..`. The first character which when parsed as JavaScript causes this error. Is the project public by any chance?
